cfripper/cli.py

```diff
@@ -154,6 +159,11 @@ def process_template(
 @click.option(
     "--rules-config-file", type=click.File("r"), help="Loads rules configuration file (type: [.py, .pyc])",
 )
+@click.option(
+    "--rules-filters-folder",
+    type=click.Path(exists=True, resolve_path=True, readable=True, file_okay=False),
+    help="Loads rules filters files from folder (type: [.py, .pyc])",
```

Comment:
The additional type hint here may be confusing if it is expecting a path of a directory. It can probably be removed or updated to say all files in the folder must be of type `.py` or `.pyc`.